lib/manageiq/active_record_connector.rb

```diff
@@ -0,0 +1,58 @@
+require "active_record"
+unless defined? DatabaseConfigurationPatch
+  require_relative "../patches/database_configuration_patch.rb"
+end
+
+module ManageIQ
+  class ActiveRecordConnector
+    # Helper class for getting the database config without needing Rails
+    #
+    # Makes use of the database_configuration_patch for the most part for
+    # grabbing this from the yaml file in config/database.yml
+    class ConnectionConfig
+      # Set a default method for handling super call in
+      # lib/patches/database_configuration_patch.rb
+      def self.database_configuration
+        if ENV["DATABASE_URL"]
+          {}
+        end
+      end
+
+      class << self
+        prepend DatabaseConfigurationPatch
+      end
+
+      def self.[](env)
+        database_configuration[env]
+      end
+    end
+
+    # Returns true if an ActiveRecord::Base.connection_config is present,
+    # otherwise returns false
+    def self.connection_exists?
+      true if ActiveRecord::Base.connection_config
```

Comment:
@Fryguy I think what you have there basically does the what I suggested above, and I decided that it wasn't much different from what is already being done, if not slightly more verbose.

I think what @jrafanie's original intent was that `.connected?` seemed like a more idiomatic way in `ActiveRecord` to check if we are connected to that database, and my explanation above is why it isn't that simple for our use case here.